problems/2704_toBe_or_notToBe/compare-functions.test.js

```diff
@@ -0,0 +1,79 @@
+const { expectFunc } = require('./to be or not to be');
+
+describe('toBe(val) accepts another value and returns true if the two values === each other. ' +
+    'If they are not equal, it should throw an error "Not Equal".', () => {
+        it.each([
+            {
+                actualVal: 5,
+                expectedVal: 5,
+                resultValue: true,
+                message: 'toBe(val) should return true: &actualVal === $expectedVal'
+            },
+            {
+                actualVal: null,
+                expectedVal: null,
+                resultValue: true,
+                message: 'toBe(val) should return true: &actualVal === $expectedVal'
+            },
+            {
+                actualVal: 1,
+                expectedVal: true,
+                resultValue: false,
+                message: 'toBe(val) should return false: &actualVal !== $expectedVal'
+            },
+            {
+                actualVal: 0,
+                expectedVal: null,
+                resultValue: false,
+                message: 'toBe(val) should return false: &actualVal !== $expectedVal'
+            }
+        ])('$message', ({ actualVal, expectedVal, resultValue }) => {
```

Comment:
```suggestion
        ])('toBe(val) should return $resultValue: $actualVal !== $expectedVal', ({ actualVal, expectedVal, resultValue }) => {
```

тогда `message` не нужен